src/mapgen_functions.cpp

```diff
@@ -537,6 +537,9 @@ void mapgen_forest_general(map *m, oter_id terrain_type, mapgendata dat, int tur
                         { 350, t_tree_apricot },
```

Comment:
(not this line, the line above all trees) `std::array<std::pair<int, ter_id>, 10> tree_chances` - that 10 is the number of objects (pairs) in the array.